app/views/pet/index.html.erb

```diff
@@ -1,14 +1,30 @@
-<div class="content">
```

Comment:
Can you switch this back to 2 spaces for tabs as opposed to the 500 you seem to be using here? 😆